src/firebase/firestore-methods.jsx

```diff
@@ -0,0 +1,175 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import {
+  doc,
+  writeBatch,
+  collection,
+  addDoc,
+  serverTimestamp,
+  updateDoc,
+  getDocs,
+  getDoc,
+  increment,
+  deleteDoc,
+  arrayUnion,
+  arrayRemove,
+} from "firebase/firestore";
+import { db } from "./firebase-config";
+const short = require("short-uuid");
+
+const createUser = async (firstName, lastName, email, userID) => {
+  const batch = writeBatch(db);
+  try {
+    const userRef = doc(db, userID, "userData");
+    batch.set(userRef, {
+      firstName,
+      lastName,
+      email,
+      bio: "",
+      dp: "",
+      website: "",
+    });
+    const postRef = doc(db, userID, "posts");
+    batch.set(postRef, { posts: [] });
```

Comment:
Nice to see the use of firebase's method here to write multiple data in a single query

Reply:
Thanks